src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -83,4 +83,12 @@ def load_arguments(self, _):
         c.argument('replication_name', help='The name of the replication. Default to the location name.', completer=None)
 
     with self.argument_context('acr build') as c:
-        c.argument('build_id', help='The unique build identifier.')
\ No newline at end of file
+        c.argument('build_id', help='The unique build identifier.')
+        c.argument('registry_name', options_list=['--name', '-n'], help='The name of the container registry. You can configure the default registry name using `az configure --defaults acr=<registry name>`', completer=get_resource_name_completion_list(REGISTRY_RESOURCE_TYPE), configured_default='acr')
+        c.argument('image_name', options_list=['--tag', '-t'], help="The image name and optionally a tag in the 'name:tag' format.")
```

Comment:
`--tag` is already used to refer to tag (e.g., latest). `image_name` by definition is `repository:tag`. `-t` is tenant, should avoid using it on a different parameter.

Reply:
For this particular command, we want to follow docker build convention.

Reply:
Not sure about this one. `--tag` definitely conflicts with existing parameters. Does docker `-t` mean `--tag` or `--image-name` or something else? You might be able to use `-t` since although it's generally tenant it's not used anywhere under `acr` group.

Reply:
We are aligning with docker on this. so -t is ok.

---
In reply to: [168378359](https://github.com/AppPlatPerf/azure-cli/pull/60#discussion_r168378359) [](ancestors = 168378359)